frontend/src/features/nextcloud_integration/api/getImages.ts

```diff
@@ -26,7 +28,72 @@ export const getImageList = async (path: string): Promise<Array<string>> => {
   }
 };
 
-// NOTE: Leaving this for now as it could be useful as an exmample
+/**
+ * get list of available images at the public share path from our Nextcloud
```

Comment:
Are there any constraints such a public share path needs to fulfill? E.g. is following correct?

```suggestion
 * Get list of available images from a Nextcloud public share path containing several top-level images and no other content.
```

Reply:
The public share is only the entry point. So you can also access images in sub-directories.